cfn/configs/klaxon/dev/us-east-1/config.yml

```diff
@@ -27,7 +27,7 @@ _globals:
   - !_join &CLUSTER_NAME ["-", *CLUSTER_PREFIX, *ENVIRONMENT]
   - !_join &STACK_NAME ["-", *CLUSTER_NAME, *SERVICE_NAME]
 
-template: v1/cfn/shared/apps/ecs-v2/service.template.yml
+template: cfn/templates/service.template.yml
```

Comment:
confirmed with stackjack build that this is the correct way to reference a local template